std/experimental/logger/core.d

```diff
@@ -1523,9 +1524,9 @@ class StdForwardLogger : Logger
         this.fatalHandler = delegate() {};
     }
 
-    override protected void writeLogMsg(ref LogEntry payload)
+    override protected void writeLogMsg(ref LogEntry payload) @trusted
     {
-          sharedLog.forwardMsg(payload);
+          (cast()sharedLog).forwardMsg(payload);
```

Comment:
This looks like a thread-safety violation. You're calling a non-shared method on a shared object without synchronization.